insights/specs/default.py

```diff
@@ -643,8 +644,14 @@ def sap_sid(broker):
     selinux_config = simple_file("/etc/selinux/config")
     sestatus = simple_command("/usr/sbin/sestatus -b")
     setup_named_chroot = simple_file("/usr/libexec/setup-named-chroot.sh")
-    smartctl_block = listdir("/sys/class/block", context=HostContext)
-    smartctl_l_scterc = foreach_execute(smartctl_block, "/sbin/smartctl -l scterc %s")
+
+    @datasource(LSBlock)
+    def hard_disk_list(broker):
```

Comment:
Please add a docstring to this function for the documentation.  See other datasources for examples.